src/components/dialogmote/referat/Buttonrow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+import { Flatknapp, Hovedknapp, Knapp } from "nav-frontend-knapper";
+
+const texts = {
+  send: "Lagre og send",
+  preview: "Se forhåndsvisning",
+  abort: "Avbryt",
+};
+
+interface KnapperadProps {
+  sendMethod: () => void;
+  previewMethod: () => void;
+  abortMethod: () => void;
+}
+
+const KnapperadWrapper = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  padding-top: 1em;
+  flex-wrap: wrap;
+
+  > * {
+    margin-right: 0.5em;
+    margin-left: 0;
+  }
+`;
+
+const Buttonrow = ({
+  sendMethod,
+  previewMethod,
+  abortMethod,
+}: KnapperadProps) => {
+  return (
+    <KnapperadWrapper>
```

Comment:
La til to komponenter _FlexRow_ og _FlexColumn_  i layout.tsx som jeg tenkte vi kunne bruke til å standardisere rader, kolonner og avstander mellom ting. Ikke sikkert det gir mening her, men du kan jo sjekke det ut i tilfelle det funker til ditt formål :)

Reply:
Kan se på det 👍